### PR TITLE
feat: Recipe List Render

### PR DESCRIPTION
**Pull request by DexHarris**
>> all requests checked

### CODE REVIEW

File: components/recipesList/recipesList.tsx [hunk 1]
```diff
@@ -1,9 +1,25 @@
 import React from 'react'
 
 const RecipesList = () => {
+
+  const RecipeArrayList = [
```
Comment (moncapitaine) [inline components/recipesList/recipesList.tsx:5]:
```suggestion
  const recipes = [
```

File: components/recipesList/recipesList.tsx [hunk 2]
```diff
@@ -1,9 +1,25 @@
...
+    { name: 'Börek mit Käse und Spinat', effort: '2 Stunden', }
+    { name: 'Georgische Hatschapuri', effort: '3 Stunden', }
+    { name: 'Krautwickel mit Reis und Hackfleisch', effort: '4 Stunden', }
+
+  ]
+
   return (
     <div>
-      Hier ist die Liste der Rezepte
+      Hier ist die Liste der Rezepte:
```
Comment (moncapitaine) [inline components/recipesList/recipesList.tsx:17]:
```suggestion
      <h2>Hier ist die Liste der Rezepte:</h2>
```

File: components/recipesList/recipesList.tsx [hunk 3]
```diff
@@ -1,9 +1,25 @@
...
+      <ol>
+        {RecipeArrayList.map((recipe) =>
```
Comment (Bhoft) [inline components/recipesList/recipesList.tsx:19]:
```suggestion
        {RecipeArrayList.map((recipe, index) =>
       
```

add index to map list

File: components/recipesList/recipesList.tsx [hunk 4]
```diff
@@ -1,9 +1,25 @@
...
+          <li><h2>{recipe.name}</h2>{recipe.effort}</li>
```
Comment (Bhoft) [inline components/recipesList/recipesList.tsx:20]:
```suggestion
          <li key={index}><h2>{recipe.name}</h2>{recipe.effort}</li>
```

use index as key

**Review by Bhoft — APPROVED** [review]

File: components/recipesList/recipesList.tsx (hunk 1 above)
Comment (DexHarris) [reply components/recipesList/recipesList.tsx]: done

File: components/recipesList/recipesList.tsx (hunk 2 above)
Comment (DexHarris) [reply components/recipesList/recipesList.tsx]: done

File: components/recipesList/recipesList.tsx (hunk 3 above)
Comment (DexHarris) [reply components/recipesList/recipesList.tsx]: done

File: components/recipesList/recipesList.tsx (hunk 4 above)
Comment (DexHarris) [reply components/recipesList/recipesList.tsx]: done

**Review by moncapitaine — APPROVED** [review]

**Pull request merged**